A32NX/layout.json

```diff
@@ -1,644 +1,644 @@
 {
```

Comment:
We have a CI job to edit this file, no need to change it manually. Please revert it as it pollutes the diff log ;)